guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc

```diff
@@ -11,4 +11,8 @@ ifeval::["{build}" != "satellite"]
 * If you plan to provision hosts from {Project} or {SmartProxy}, you must install {Project} and {SmartProxies} on a system with `grub2` version 2.05 or higher or system with fixes for HTTP Boot, for example CentOS version 7.9 or 8.3 or higher. For other operating systems, copy the newest GRUB build to `/var/lib/tftpboot/grub2/grubx64.efi`.
 endif::[]
 
-* You can install {Project} and {SmartProxies} in IPv6-only systems, dual-stack installation not supported.
+* You must deploy an external DHCP IPv6 server as a separate unmanaged service to bootstrap clients into GRUB2, which then configures IPv6 networking either using DHCPv6 or or assigning static IPv6 address. This is required because the DHCP server in {RHEL} (ISC DHCP) does not provide an integration API for managing IPv6 records, therefore the {SmartProxy} DHCP plug-in that provides DHCP management is limited to IPv4 subnets.
+
+* You must deploy an external IPv4 HTTP proxy server. This is required because {Project} distributes content only over IPv4 networks, therefore you must use an IPv4 proxy to redirect that content to hosts in your IPv6 network.
+
+* You must configure {Project} to use this IPv4 HTTP proxy server as the default proxy. For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.8-beta/html/installing_satellite_server_from_a_connected_network/performing-additional-configuration#adding-a-default-http-proxy_satellite[Adding a Default HTTP Proxy to Satellite].
```

Comment:
Add a sentence that shows up when "foreman" build is done: This is only relevant for Katello content.

Reply:
Done